### PR TITLE
Add snippets for Media CDN - Dual Token Feature

### PR DESCRIPTION
**Pull request by msampathkumar**
## Description

Add new Python snippets for Media CDN - Dual Token

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

File: media_cdn/dualtoken.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        full_path:  A full path to sign, starting with the first '/'.
+                    For example: /path/to/content.mp4
+        path_globs: a set of ','- or '!'-delimited path glob strings.
+                    For example: /tv/*!/film/* to sign paths starting with /tv/ or /film/ in any URL.
+        session_id: a unique identifier for the session
+        data: data payload to include in the token
+        headers: header name and value to include in the signed token in name=value format.  May be specified more than once.
+                    For example: [{'name': 'foo', 'value': 'bar'}, {'name': 'baz', 'value': 'qux'}]
+        ip_ranges: A list of comma separated ip ranges. Both IPv4 and IPv6 ranges are acceptable.
+                    For example: b"203.0.113.0/24,2001:db8:4a7f:a732/64"
```
Comment (justin-mp) [inline media_cdn/dualtoken.py:69]: ip_ranges is pytyped to be str, so you don't want the `b` prefix in the example.

**Review by justin-mp — APPROVED** [review]

File: media_cdn/dualtoken.py (hunk 1 above)
Comment (msampathkumar) [reply media_cdn/dualtoken.py:69]: Got it! Many thanks for reporting it.

Comment (msampathkumar) [reply media_cdn/dualtoken.py]: Update the code now.

File: media_cdn/dualtoken.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+#
+# Copyright 2022 Google, Inc.
```
Comment (m-strzelczyk) [inline media_cdn/dualtoken.py:3]: It's 2023. Don't live in the past!

**Review by m-strzelczyk — APPROVED** [review]

**Comment by m-strzelczyk** [comment]
You can ignore the 3.11 version failing. It's still misconfigured.

File: media_cdn/dualtoken.py (hunk 2 above)
Comment (msampathkumar) [reply media_cdn/dualtoken.py:3]: Hi @m-strzelczyk, unfortunately I did not notice this earlier. I make a note of this in my next PR. Thanks

**Pull request merged**